requirements/requirements-dev.in

```diff
@@ -3,3 +3,6 @@
 jupyter_contrib_nbextensions
 jupyter_nbextensions_configurator
 black
+nbsphinx
+recommonmark
+sphinx_rtd_theme
```

Comment:
Is a newline necessary at the end of the file?

Reply:
I don't think it really is necessary, but I feel its appropriate.

Reply:
Not really. I asked because github indicated with a red circle that there's no newline at the end of the file. The installation worked regardless.

Reply:
Ah yes, now I got it. Resolved in 47c3d8562e38577d58bfe45b34c7cfcdabe94242.